### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by DinuWije**
## Functionality:
Currently logs a message to console and redirects the user to /admin endpoint when Google Sign In button is clicked

## Todo:
- Add API call to backend to log the user in when the button is clicked
- Edit CSS once the Figma designs are hi-fi
## /login endpoint: 
image

## /unauthorized endpoint:
image

### CODE REVIEW

**Review by AbhiPrasad — APPROVED** [review]
sick, good stuff.

Plz run `yarn fix` before merge!

File: ui/src/pages/Login.tsx [hunk 1]
```diff
@@ -1,14 +1,76 @@
...
+  position: absolute;
+`;
+
+const StyledCard = styled(Card)`
+  max-height: 35vh;
+  max-width: 25vw;
+`;
+
+const TitleText = styled.p`
+  font-size: 20px;
```
Comment (hanlinc27) [inline ui/src/pages/Login.tsx]: can you move the TitleText and Message Text to the typography file? And then use fontSize.whicheverText for consistency.

Comment (hanlinc27) [reply ui/src/pages/Login.tsx]: Wait nvm I realized this is all lofi so the font sizes are still in the air - would be good though to still keep the message and title texts in the typography file like the other components :)

File: ui/src/pages/Login.tsx [hunk 2]
```diff
@@ -1,14 +1,76 @@
 import * as React from "react";
 import { Redirect } from "react-router-dom";
```
Comment (hanlinc27) [inline ui/src/pages/Login.tsx]: run `yarn run fix`

File: ui/src/App.tsx [hunk 3]
```diff
@@ -26,8 +26,11 @@ function App(): JSX.Element {
             <Route exact path="/">
               <ShoeMap />
             </Route>
-            <Route path="/login">
-              <Login />
+            <Route exact path="/login">
+              <Login login={true} />
```
Comment (hanlinc27) [inline ui/src/App.tsx]: it will have a value of **true** by default so you can just put **login** instead of explicitly defining

**Review by hanlinc27 — CHANGES_REQUESTED** [review]

File: ui/src/pages/Login.tsx [hunk 4]
```diff
@@ -23,29 +22,22 @@ const StyledCard = styled(Card)`
...
   text-align: center;
   margin-bottom: 5vh;
 `;
 
-export const Login: React.FC<{ login: boolean }> = ({ login }) => {
+interface LoginProps {
+  login: boolean;
+}
+
+export const Login: React.FC<LoginProps> = (props) => {
```
Comment (AbhiPrasad) [inline ui/src/pages/Login.tsx]: let's just destructure the props here

**Review by hanlinc27 — APPROVED** [review]

**Pull request merged**